### DISCUSSION TITLE
Nginx Proxy Manager LXC update procedure?

### DISCUSSION THREAD

**Discussion by vsc55** [tteck/Proxmox#2960] [Q&A] [answered]
Hi,
I currently have Nginx Proxy Manager running on a docker but I want to migrate it to an LXC.
I have Nginx Proxy Manager configured using the json file as a database.
Before making the change from Docker to LXC, I would like to know what the procedure is to update LXC to the latest version of Nginx Proxy Manager without losing the database.
Does the script itself create a new LXC each time? Does the script update the LXC that is created without losing the settings? Do I have to do any manual file export procedure?
Thanks.

**Comment by tteck** [comment, marked as answer]
Type the command `update` in the LXC console

**Comment by vsc55** [comment]
Thank you very much @tteck for the speed.

**Comment by danluckner** [comment]
I'm on the same situation as @vsc55, trying to migrate from docker to LXC container.
I used the helper script to spin up the LXC, then stopped services like nodejs etc, copied the files from the docker nginx to the new LXC and restarted, but no to avail.
These are following directories I used to copy the files from:

```
Docker  -------- LXC
/data -------- /data
/letsencrypt -------- /etc/letsencrypt
```
At this point, I'm not sure what I'm doing wrong. 
I've consulted this guide as well: https://github.com/NginxProxyManager/nginx-proxy-manager/discussions/1529
